app/controllers/placements_controller.rb

```diff
@@ -21,12 +74,43 @@ def create
   # service_id: params[:placement][:service_id],
 
   def index
-    @placements = Placement.all
-    search = "%#{params[:search]}%"
-    @placements_search = Placement.where('name LIKE ? OR county LIKE ?',
-                                         search, search)
+
+
+    # search = "%#{params[:search]}%"
+    # @placements = Placement.where('name LIKE ? OR county LIKE ?', search, search)
+      
+    @q = Placement.ransack(params[:q])
```

Comment:
q is not a good choice for a variable name.  You want to use something descriptive.